com.github.tralph3.Steam_Metadata_Editor.yml

```diff
@@ -0,0 +1,51 @@
+app-id: com.github.tralph3.Steam_Metadata_Editor
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: start.sh
+modules:
+  - name: Steam_Metadata_Editor
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/tralph3/Steam-Metadata-Editor
+        commit: 96265afc5179304514d78091a456b138b0a44e34
+    build-commands:
+      - install -dm755 /app/share/Steam-Metadata-Editor
+      - cp -r src/. /app/share/Steam-Metadata-Editor
+      - echo -e "#!/bin/bash\npython3 /app/share/Steam-Metadata-Editor/main.py \$@" > start.sh
```

Comment:
you should make this a `shell` source.

Reply:
Should I not use git, and just clone it with a standard git command if I use a shell source?

Reply:
I meant `script` source.

![image](https://user-images.githubusercontent.com/114441/224573951-84083df0-6338-4062-97a8-09f5a7cbeb7c.png)